airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -452,6 +460,11 @@ def _check_success_task(self) -> None:
         if len(response.get('failures', [])) > 0:
             raise AirflowException(response)
 
+        if self.task_log_fetcher and not self.task_log_fetcher.is_log_stream_created():
+            raise AirflowException(
+                'Cloudwatch log stream for the task is not found. The task probably failed to start.'
+            )
+
         for task in response['tasks']:
```

Comment:
```suggestion
        for task in response['tasks']:
            if task.get('stopCode', '') == 'TaskFailedToStart':
                raise AirflowException('The task failed to start.')
```

Such check is probably better and simpler. And no need to check cloudwatch log streams.

Reply:
how about
```python
raise AirflowException('The task failed to start due to: {}'.format(task.get('stoppedReason', '')))
```